src/Libraries/Microsoft.Extensions.Compliance.Redaction/RedactionExtensions.cs

```diff
@@ -0,0 +1,68 @@
+
+/// <summary>
+/// Extensions to configure specific redactors.
+/// </summary>
+public static class RedactionExtensions
+{
+    /// <summary>
+    /// Sets the HMAC redactor to use for a set of data classes.
+    /// </summary>
+    /// <param name="builder">The builder to attach the redactor to.</param>
+    /// <param name="configure">Configuration function.</param>
+    /// <param name="classifications">The data classes for which the redactor type should be used.</param>
+    /// <returns>The value of <paramref name="builder" />.</returns>
+    /// <exception cref="ArgumentNullException"><paramref name="builder"/>, <paramref name="configure" /> or <paramref name="classifications" /> are <see langword="null"/>.</exception>
+    [Experimental(diagnosticId: Experiments.Compliance, UrlFormat = Experiments.UrlFormat)]
+    public static IRedactionBuilder SetHmacRedactor(this IRedactionBuilder builder, Action<HmacRedactorOptions> configure, params DataClassification[] classifications)
+    {
+        _ = Throw.IfNull(builder);
+        _ = Throw.IfNull(configure);
+        _ = Throw.IfNull(classifications);
+
+        _ = builder
+                .Services
+                .AddOptionsWithValidateOnStart<HmacRedactorOptions, HmacRedactorOptionsValidator>()
+                .Configure(configure);
+
+        return builder.SetRedactor<HmacRedactor>(classifications);
+    }
+
+    /// <summary>
+    /// Sets the HMAC redactor to use for a set of data classes.
+    /// </summary>
+    /// <param name="builder">The builder to attach the redactor to.</param>
+    /// <param name="section">Configuration section.</param>
+    /// <param name="classifications">The data classes for which the redactor type should be used.</param>
+    /// <returns>The value of <paramref name="builder" />.</returns>
+    /// <exception cref="ArgumentNullException"><paramref name="builder"/>, <paramref name="section" /> or <paramref name="classifications" /> are <see langword="null"/>.</exception>
+    [Experimental(diagnosticId: Experiments.Compliance, UrlFormat = Experiments.UrlFormat)]
+    [DynamicDependency(DynamicallyAccessedMemberTypes.All, typeof(HmacRedactorOptions))]
+    [UnconditionalSuppressMessage(
+        "Trimming",
+        "IL2026:Members annotated with 'RequiresUnreferencedCodeAttribute' require dynamic access otherwise can break functionality when trimming application code",
+        Justification = "Addressed with [DynamicDependency]")]
+    public static IRedactionBuilder SetHmacRedactor(this IRedactionBuilder builder, IConfigurationSection section, params DataClassification[] classifications)
+    {
+        _ = Throw.IfNull(builder);
+        _ = Throw.IfNull(section);
+        _ = Throw.IfNull(classifications);
+
+        _ = builder
+                .Services.AddOptionsWithValidateOnStart<HmacRedactorOptions, HmacRedactorOptionsValidator>()
+                .Services.Configure<HmacRedactorOptions>(section);
```

Comment:
```suggestion
                .Bind(section);
```

Reply:
None of the existing redactors in R9 call Bind here. Is this really needed?

Reply:
None of the existing R9 redactor extensions call Bind in this case. Is this really necessary?

Reply:
No, under the hood it does the same thing. So it's up to you

Reply:
I'll leave it out then.